CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#6256](https://github.com/rubocop-hq/rubocop/pull/6256) Fix false positive in `Naming/FileName`. ([@sinsoku][])
```

Comment:
CI has failed. This entry forgets a colon.
https://travis-ci.org/rubocop-hq/rubocop/jobs/426512939#L1202-L1211

```diff
- * [#6256](https://github.com/rubocop-hq/rubocop/pull/6256) Fix false positive in `Naming/FileName`. ([@sinsoku][])
+ * [#6256](https://github.com/rubocop-hq/rubocop/pull/6256): Fix false positive in `Naming/FileName`. ([@sinsoku][])
```

Reply:
Thanks for a review. I fixed it 👍